stage_descriptions/base-07-mg5.md

```diff
@@ -8,6 +8,10 @@ For example, if PATH is `/dir1:/dir2:/dir3`, your shell should search in `/dir1`
 - If a matching files is found and it has execute permissions, your shell should print the path to the file. 
 - If no matching files are found, your shell should print `<command>: not found`.
 
+{{#lang_is_elixir}}
```

Comment:
1. Let's move this to the notes section
2. Let's add a fallback case for a generic language
3. We don't have to say it's not recommended - a shell would usually use this, so it's okay. 

For Elixir, you could say something like: 

> You can either use [System.find_executable]() or implement the PATH traversal logic on your own. 

For others, could be something like: 

> Most programming languages have methods in their standard library that can help with PATH traversal. For example, [xyz]() in Python and []() in Go.